buildpack/telemetry/splunk.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import os
+from buildpack import util
+
+
+SPLUNK_ENV_VARS = {
+    "SPLUNK_PORT": os.getenv("SPLUNK_PORT", default="8088"),
+}
+
+REQUIRED_SPLUNK_ENV_VARS = ["SPLUNK_HOST", "SPLUNK_TOKEN"]
+
+
+def _set_default_env(m2ee):
+    for var_name, value in SPLUNK_ENV_VARS.items():
+        util.upsert_custom_environment_variable(m2ee, var_name, value)
+
+
+def stage():
+
+    if not is_splunk_enabled():
+        return
+
+    logging.info("Splunk is configured.")
```

Comment:
Shouldn't this line go after the `update_config()` is invoked? That's where we configure the required vars.

Reply:
Log line changed